operator/pkg/controller/virtualippool/controller.go

```diff
@@ -303,6 +297,22 @@ func (r *VirtualIPPoolReconciler) mutateConfigMap(obj *corev1.ConfigMap) error {
 
 	desired := &HLConfig{}
 	desired.Init()
+
+	if r.instance.Spec.Healthcheck != nil {
+		healthInfo := r.instance.Spec.Healthcheck.HttpGet
+		ip := healthInfo.IP
+		if ip == "" {
+			ip = "{NODE_IP}"
+		}
+		desired.Healthcheck = fmt.Sprintf(
+			"%s://%s:%d%s",
+			strings.ToLower(healthInfo.Scheme),
+			ip,
+			healthInfo.Port,
+			healthInfo.Path,
+		)
```

Comment:
Since `healthInfo.Path` is still a free-form string, we should be more careful... IMO, using [`net/url`](https://pkg.go.dev/net/url) could be a better approach, since we have all the parts we need to build a `URL` instance. The only nuance would be about using the `{NODE_IP}` placeholder, but we could work around this by using a reserved hostname, for instance, and replacing it with the desired environment variable in our Jinja template.

Reply:
Right ok

Reply:
Fixed in https://github.com/scality/metalk8s/pull/4000/commits/be84bb1cd7b81a1695c898f4426b1891a415c3b9